plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -70,7 +70,7 @@ func defaultPriorities() util.StringSet {
 			"ServiceSpreadingPriority",
 			factory.PriorityConfigFactory{
 				Function: func(args factory.PluginFactoryArgs) algorithm.PriorityFunction {
-					return priorities.NewServiceSpreadPriority(args.ServiceLister)
+					return priorities.NewSelectorSpreadPriority(args.ServiceLister, args.ControllerLister)
```

Comment:
On line 70 you should change ServiceSpreadingPriority -> SelectorSpreadingPriority